controls/togglebutton/visual-structure.md

```diff
@@ -1,34 +1,30 @@
 ---
 title: Visual Structure
 page_title: .NET MAUI ToggleButton Documentation - ToggleButton Visual Structure
-description: Learn what visual elements are displayed by the Telerik UI for .NET MAUI ToggleButton, and see how these elements build the visual structure of the control.
+description: Learn what visual elements are displayed by the Telerik UI for .NET MAUI ToggleButton, and see how these elements build the visual structure of the component.
 position: 1
 slug: togglebutton-visual-structure
 ---
 
 # .NET MAUI ToggleButton Visual Structure
 
-The visual structure of the .NET MAUI ToggleButton represents the anatomy of the UI component. Being familiar with the visual elements of the Calendar allows you to quickly find the information required to configure them.
+The visual structure of the .NET MAUI ToggleButton represents the anatomy of the UI component. Being familiar with the visual elements of the ToggleButton allows you to quickly find the information required to configure them.
 
 The following image shows the anatomy of the ToggleButton.
 
 ![.NET MAUI ToggleButton Visual Structure](images/togglebutton-visual-structure.png "Visual elements of ToggleButton control")
 
 ## Displayed Elements
 
-## Additional Resources
+* **Content**&mdash;The content of the ToggleButton.
```

Comment:
Same note as for TemplatedButton.

Reply:
the structure from the link is applicable for button control, here the content could be anything, there isn't an icon that can be explicitly set through property. so the element inside the button is only its content (the content can be anything)